io.github.junalmeida.webapps_manager.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "io.github.junalmeida.webapps_manager",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.PySide.BaseApp",
+    "base-version": "6.7",
+    "command": "webapps_manager",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=host-os",
+        "--filesystem=/var/lib/flatpak",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Flatpak"
```

Comment:
this needs an exception. why do you need it?

Reply:
The app needs to call `flatpak-spawn --host` to launch the browser command to test. It also needs to create `.desktop` files in user's home.

Reply:
there is no need to do that, use the default browser. there is at least two other webapp managers that don't need flatpak-spawn https://github.com/flathub/io.github.elevenhsoft.WebApps https://github.com/flathub/net.codelogistics.webapps

Reply:
The purpose of the app is not to use the default browser. The purpose is to create desktop-like web apps using specific browsers.
The app I'm proposing does not use a webview own window like the one from codelogistics.
It uses a regular browser that is installed either by host system packages or other flatpaks more like the Quick Apps from elevenhsoft, however this one doesn't have a Launch command to test a recently created webapp right away.